content/news/2022-11-24-ESG-WP5/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Work Package 5 of the EuroScienceGateway project
+date: "2022-11-24"
+tease: "WP5 - a showcase results from these 3 early adopters to embrace Open Science best practices."
+hide_tease: false
+tags: [esg-wp5, esg]
+subsites: [all-eu, esg]
+main_subsite: eu
+---
+
+The fifth Work Package (WP5) of the EuroScienceGateway project will demonstrate the benefits of the EuroScienceGateway services with 3 communities,
+namely (i) Biodiversity and Climate Science with the impact on human health, (ii) Materials science and (iii)
+Astrophysics. WP5 will showcase results from these 3 early adopters that will lead by example to embrace
+Open Science best practices. Each of the adopters are at a different level of maturity in their adoption of
+Galaxy for their communities with Biodiversity/Climate being the most experienced and Astrophysics the
+least experienced. Together with WP1 (in particular T1.6), WP5 will support the early adopters as well as
+new communities to get the most out of the framework described in this proposal and extend it if needed.
+WP5 will ensure the adoption of EuroScience Gateway services.
+Discussions between hardware, software, and application developers will take place to ensure the
+community-driven nature of EuroScienceGateway. WP5 will endeavour to support other technical work
+packages to provide feedback from the field researchers (about their usage and workflow compositions) in an
+effective and relevant way.
+KPIs will be derived and monitored for i) number and usage of tools and workflows developed by the
+communities, ii) number of data sources accessible via Galaxy and iii) the number and usage of Pulsar nodes
+(Bring Your Own Compute & Bring Your Own Storage).
```

Comment:
Since we do not explain this here I would also remove "(Bring Your Own Compute & Bring Your Own Storage)"